docs/synapse/userguides/userguide_section7.rst

```diff
@@ -0,0 +1,183 @@
+
+In cases where these paresed secondary properties are themselves forms, Synapse will also automatically create nodes for those forms if they do not already exist. For example, for an ``inet:fqdn`` node, secondary properties include the ``inet:fqdn:host`` (the hostname of the fqdn, stored as a string) and ``inet:fqdn:domain`` (the parent fqdn of the fqdn, which is also an ``inet:fqdn`` form.). If you create the node ``inet:fqdn=foo.woot.com``, Synapse will create:
+
+- an ``inet:fqdn`` node for ``foo.woot.com``.
+- secondary proeprties ``inet:fqdn:host=foo`` and ``inet:fqdn:domain=woot.com`` (these do not need to be manually defined when creating the node).
+- an additional ``inet:fqdn`` node for ``woot.com``, if it does not already exist in the Cortex.
+
+Examples of simple nodes include:
+
+- **IP addresses.** An IP address (IPv4 or IPv6) must be unique within its address space and can be defined by the address itself (``inet:ipv4=1.2.3.4``). Secondary properties (depending on need) may include things like the associated Autonomous System Number (ASN), geolocation data, and whether the IP belongs to a specialized or reserved group (e.g., private, multicast, etc.)
+
+- **Domains.** Similarly, an Internet domain must be globally unique (the same domain cannot be registered by two different entities) and can be defined by the domain string (``inet:fqdn=woot.com``). Additional secondary properties may include whether the domain is a top-level domain (TLD).
+
+- **Files.** Files are unique based on their content. Instead of doing a byte-by-byte comparison of content, files are typically "compared" by generating a hash of the file and comparing the hash of one file with the hash of another file; if the hashes match, the files are considered to be the same. However, improvements in both cryptanalysis and computational power have made hash collisions more likely – that is, it is possible to not only find but in some cases to deliberately generate two files with different content that nonetheless result in the same hash. Thus, some hash algorithms (MD5, and increasingly SHA1) that were once considered unique no longer are, and there is no guarantee that this trend will not eventually hold true for additional hash algorithms. Because no hash can be considered reliably unique, the primary property of a file is represented by a GUID (``file:bytes=<guid>``). Specifically, if known, Synapse stores four hashes (MD5, SHA1, SHA256, SHA512) as secondary properties and generates a stable (deterministic) GUID from those hashes. For this reason a Synapse file:bytes GUID is sometimes referred to as a “superhash”.
```

Comment:
The last reference to `file:bytes` in this section creates a link to a local file called bytes.